src/uct/ib/base/ib_iface.c

```diff
@@ -156,11 +162,18 @@ ucs_status_t uct_ib_iface_recv_mpool_init(uct_ib_iface_t *iface,
                         config->rx.mp.max_bufs);
     }
 
+    if (!is_accel) {
```

Comment:
need to rename the is_accel parameter to express what it does and not how it's used
also, need to calculate grow only if needed, and prefer `if (x) else ` over `if (!x) else` if there is no importance for performance